### PR TITLE
SDI-265: ♻️  Convert notifications to nunjucks

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: server/controllers/notificationController.ts [hunk 1]
```diff
@@ -111,11 +106,14 @@ export default class NotificationController {
...
-    await this.notificationService.updateSnooze(token, this.getFormattedFutureDate(pauseUnit, pauseValue))
+    await this.notificationService.updateSnooze(
+      token,
+      NotificationController.getFormattedFutureDate(pauseUnit, pauseValue),
+    )
     return res.redirect('/notifications/manage')
   }
 
-  private getFormattedFutureDate(pauseUnit: string, pauseValue: number) {
+  private static getFormattedFutureDate(pauseUnit: string, pauseValue: number) {
```
Comment (petergphillips) [inline server/controllers/notificationController.ts:116]: intellij suggestion

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -37,12 +39,6 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
...
 
-  njkEnv.addFilter('initialiseName', (fullName: string) => {
-    // this check is for the authError page
-    if (!fullName) {
-      return null
-    }
-    const array = fullName.split(' ')
-    return `${array[0][0]}. ${array.reverse()[0]}`
-  })
+  njkEnv.addFilter('initialiseName', initialiseName)
```
Comment (petergphillips) [inline server/utils/nunjucksSetup.ts:42]: moved to separate function so can test

File: server/views/helpers/helpers.ejs [hunk 3]
```diff
@@ -19,27 +19,6 @@
...
-  getRelativeModifiedDate = (moment, modifiedDate) => {
-
-    var s = ( moment().diff(moment(modifiedDate)))/ 1000,
-      m = s / 60,
-      h = m / 60,
-      d = h / 24,
-      y = d / 365.242199,
-      tmp;
-
-    return ( tmp = Math.round(s) ) === 1 ? 'just now'
```
Comment (petergphillips) [inline server/views/helpers/helpers.ejs:31]: switched to using `fromNow` instead rather than calculating like this

Comment (steverendell) [reply server/views/helpers/helpers.ejs:31]: 👍

**Review by steverendell — APPROVED** [review]
lgtm with suggestion

File: server/views/helpers/helpers.ejs (hunk 3 above)
Comment (petergphillips) [reply server/views/helpers/helpers.ejs:31]: yes, good point.  Will fix in future PR

**Pull request merged**